backend/src/apiserver/model/run_test.go

```diff
@@ -0,0 +1,37 @@
+package model
+
+import (
+	"fmt"
+	"testing"
+
+	sq "github.com/Masterminds/squirrel"
+	api "github.com/kubeflow/pipelines/backend/api/go_client"
+	"github.com/kubeflow/pipelines/backend/src/apiserver/list"
+	"github.com/stretchr/testify/assert"
+)
+
+// Test model name usage in sorting clause
+func TestAddStatusFilterToSelect(t *testing.T) {
+	listable := &Run{
+		UUID:           "run_id_1",
+		CreatedAtInSec: 1,
+		Name:           "run_name_1",
+		Conditions:     "Succeeded",
+	}
+	protoFilter := &api.Filter{}
+	protoFilter.Predicates = []*api.Predicate{
+		{
+			Key:   "status",
+			Op:    api.Predicate_EQUALS,
```

Comment:
nit: one more test case with NOT_EQUALS and one more Run whose condition is not Succeeded?
so that we can be sure the filter is working

Reply:
Going to added the not_equal filter. The example run condition doesn't matter here, since we don't actually query. We only verify against the sql query that is generated by the listable class.

Reply:
Sure, LGTM

Reply:
Also, in case the value is a concern. I added the assertion on generated "args" in addition to the generated sql.